core/src/main/java/org/apache/gravitino/storage/relational/mapper/provider/postgresql/OwnerMetaPostgreSQLProvider.java

```diff
@@ -116,6 +122,11 @@ public String sotDeleteOwnerRelBySchemaId(Long schemaId) {
         + FilesetMetaMapper.META_TABLE_NAME
         + " ft WHERE ft.schema_id = #{schemaId} AND "
         + "ft.fileset_id = ot.metadata_object_id AND ot.metadata_object_type = 'FILESET'"
+        + " UNION "
```

Comment:
I found the method name incorrect; it should be "soft...".

Reply:
Thanks for remind. Fixed.